packages/SwingSet/src/liveslots/liveslots.js

```diff
@@ -1350,11 +1352,7 @@ function build(
     );
 
     // here we finally invoke the vat code, and get back the root object
-    const rootObject = await buildRootObject(
-      harden(vpow),
-      harden(vatParameters),
-      baggage,
-    );
+    const rootObject = await buildRootObject(vpow, vatParameters, baggage);
```

Comment:
https://github.com/endojs/endo/blob/c5582ca7473e0a5d94ef4753ff54e0626cdb1d0a/packages/marshal/src/marshal.js#L507-L510

Reply:
https://github.com/endojs/endo/blob/c5582ca7473e0a5d94ef4753ff54e0626cdb1d0a/packages/marshal/src/marshal.js#L501

Reply:
Ah, so anything that comes back from `unserialize` lands in our hands in an already hardened state?  That makes sense.  (But wait: I thought you couldn't harden something twice, so why didn't the previous call to harden (the call you took out that I was asking about) cause an error?)

Reply:
You can always `harden` multiple times. It is fully idempotent. Redundant applications are completely harmless.

I think you're thinking of `Far`. It can only be applied one in order to prevent an attack, among other things.

Resolving.